blog/news-and-updates/team-spotlight-sdk.mdx

```diff
@@ -0,0 +1,87 @@
+---
+title: Team spotlight - SDK
+description: In this issue, we sat down with the SDK team to talk all things IC SDK and the upcoming Cycles Ledger feature! For most IC devs, the IC SDK is at the center of their workflow, so I'm sure many of you have been looking forward to this team spotlight!
+tags: [Team spotlight]
+image: /img/blog/dev-update-blog-SDK.jpg
+---
+
+Hey devs and welcome to this edition of the IC team spotlight! In this issue, we sat down with the SDK team to talk all things IC SDK and the upcoming Cycles Ledger feature! For most IC devs, the IC SDK is at the center of their workflow, so I'm sure many of you have been looking forward to this team spotlight! With that, let's dive into the interview. 
+
+**To get started, let's get a brief overview of the team. What is the purpose of the SDK team?**
+
+*To provide tools that make building on the Internet Computer fast, easy, and fun.*
+ 
+**What is the composition of the SDK team?**
+
+*The SDK team has seven members. The engineers on the SDK team are Eric Swanson, Linwei Shang, Kyle Peacock, Severin Siffert, Marcin Nowak-Liebiediew,  and Adam Spofford. The team also includes Jessie Mongeon, a technical writer, Thomas Locher, a visiting member from the research team, and Jason Ibrahim, the team’s engineering manager.*
```

Comment:
has everyone consented to using their full names? we want to respect people's privacy. should we change this to something more generic perhaps?

Reply:
Removed names and made generic 👍🏻